stats/kafka/config.go

```diff
@@ -0,0 +1,107 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2016 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package kafka
+
+import (
+	"github.com/kubernetes/helm/pkg/strvals"
+	"github.com/mitchellh/mapstructure"
+	"gopkg.in/guregu/null.v3"
+)
+
+type ConfigFields struct {
+	// Connection.
+	Brokers []null.String `json:"brokers" envconfig:"KAFKA_BROKERS"`
+
+	// Samples.
+	Topic  null.String `json:"topic" envconfig:"KAFKA_TOPIC"`
+	Format null.String `json:"format" envconfig:"KAFKA_FORMAT"`
+}
+
+// config is a duplicate of ConfigFields as we can not mapstructure.Decode into
```

Comment:
Argh, forgot about that annoying mapstructure limitation... Don't have a better solution though, besides finding something better than strvals that actually understands struct tags...